index.html

```diff
@@ -10,5 +10,13 @@
 </head>
 <body>
   <h1 class="title">Front-End Tools</h1>
+  <div class="tech">
+    <p class="text" id="titleTech">ReactJS </p>
```

Comment:
Delete the `id` attribute because isn't a good practice use this for assign styles. Use only class. And for keeping a correct syntax and structure in the HTML replace this tag for `h3`